ui.frontend/src/main/webpack/site/js/service/serviceEvent.ts

```diff
@@ -0,0 +1,26 @@
+export const onLoadMoreClick = new CustomEvent('SAAS-loadMore-click', {
```

Comment:
**Recommendation**

DRY :D

I would create one function that takes as argument the name of the event and the data.
Also I would create an object centralizing the event names. Let's keep the names in lowercase/hyphenized too.

Eg.
``` 
const events = {
  tabswitch: 'saas-tab-switch',
  searchItem: 'saas-search-item-click',
  ...
}
```

Reply:
This is done.